fundingmanager.go

```diff
@@ -422,6 +451,33 @@ func (f *fundingManager) handlePendingChannels(msg *pendingChansReq) {
 	msg.err <- nil
 }
 
+// handlePendingCloseChannels
+func (f *fundingManager) handlePendingCloseChannels(msg *pendingCloseChansReq) {
```

Comment:
This query shouldn't be routed through the `fundingManager`. Once a channel funding workflow has been completed, gating access to further state of the channel isn't the responsibility of the `fundingManager`. Instead, the query should be carried out directly within the implementation of the RPC call (wihtin the `rpcServer`).